src/LessMsi.Core/Msi/ExtractionMode.cs

```diff
@@ -1,11 +1,11 @@
-﻿namespace LessMsi.Cli
+﻿namespace LessMsi.Msi
 {
     public enum ExtractionMode
     {
         /// <summary>
-        /// Default value indicating that no extraction should be performed.
+        /// Default value indicating that a regular extraction should be performed.
         /// </summary>
-        None,
+        Default,
```

Comment:
NOTE: Changing this would be a breaking change to anyone using the library. I don't think that's a problem since this is the LessMsi.Cli library and to my knowledge only people depend on the Lessmsi.Core library. I am okay making this change, just want to point it out.